StepsToCare/CareDashboard.ascx.cs

```diff
@@ -876,6 +915,24 @@ protected void rFilter_DisplayFilterValue( object sender, GridFilter.DisplayFilt
         /// <param name="e">The e.</param>
         protected void rFollowUpFilter_DisplayFilterValue( object sender, GridFilter.DisplayFilterValueArgs e )
         {
+            if ( AvailableAttributes != null )
```

Comment:
Same code as lines 829 - 845. Should this be extracted to a method?

Reply:
@guymaness2 They are slightly different, and it is more of a standard to follow from Rock Grid's, so it was intentionally copied twice, there are other things that are similar with these grids that have not been optimized either. Because of that in this instance, I am going to choose not to, but good catch.